tests/integration/attributes/test_transaction_additional_params2.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
I would add tests passing invalid values to both forms of this API.  For example, passing `NULL` or passing an integer or object where a string is expected, etc.   Make sure a mistake calling from PHP will not take the agent down.